### ISSUE TITLE
Accordion - DS site WCAG accessibility documentation

### ISSUE THREAD

**Issue by jcfilben** [grommet/hpe-design-system#4961]
Add Accessibility information to the Accordion page on the Design System site.

- [x] Assess which [WCAG rules](https://www.w3.org/WAI/WCAG22/quickref/?versions=2.2) are applicable to the component and if they are passing/failing.
- [x] Create a pull request on the Design System site to add this information to the component page. Follow the format here: https://github.com/grommet/hpe-design-system/pull/4675/files. The finished result should look something like this: https://design-system.hpe.design/components/menu?q=menu#wcag-compliance.
- [x] Create issues in the relevant repository (grommet, grommet-theme-hpe, etc.) for any rules where we are not meeting WCAG AA or if there is something observed that it feels like we should fix. We should create 1 issue per rule failure. Issues where we are not meeting AA should be tagged with 'bug'. All issues created through this process should have the 'accessibility' label and be added to the accessibility board.

The pull request should be up on Github within the first week of the sprint so that the second week of the sprint can be focused on review and polishing.

**Comment by CillianHPE** [comment]
First pass of a11y audit for accordion added to the **[excel sheet](https://hpe.sharepoint.com/:x:/r/teams/hpe-design/Shared%20Documents/HPE%20Design%20System/Accessibility/WCAG%20Rule%20Evaluation%20Template.xlsx?d=we975e19f32644f4cbd0a446f23f89277&csf=1&web=1&e=O0AJuS&nav=MTVfezMxRDdFRkFBLUIxQkUtNDA2MS04OUZGLTVBOTQ0NkNFOTQ2RX0)**

**Comment by jcfilben** [comment]
@luketa8 can you do a first pass review on this?

**Comment by luketa8** [comment]
First pass review done in [spreadsheet](https://hpe.sharepoint.com/:x:/r/teams/hpe-design/_layouts/15/Doc.aspx?sourcedoc=%7BE975E19F-3264-4F4C-BD0A-446F23F89277%7D&file=WCAG%20Rule%20Evaluation%20Template.xlsx&action=default&mobileredirect=true)

**Comment by CillianHPE** [comment]
Issues in Backlog

https://github.com/grommet/grommet/issues/7584

**Comment by jcfilben** [comment]
PR merged, marking as closed
